main.go

```diff
@@ -47,6 +49,9 @@ func main() {
 	go func() {
 		log.Fatal(metrics.ListenAndServe(*promPort))
 	}()
+	go func() {
```

Comment:
We do want to run it on prod as well, right? I guess that should be fine, they mention it should not add more than 5% CPU usage, but just want to double-check.

Reply:
@leszko could you link to where you saw the reference to 5% cpu usage? I could be wrong but to my understanding it shouldn't add any extra cpu usage other than having an extra http listener, the actual profiling doesn't occur until you make a request to the handler to fetch some pprof data.

Reply:
https://stackoverflow.com/questions/26545159/how-big-is-pprof-import-overhead-in-go

Reply:
Check this out: https://stackoverflow.com/questions/64057727/is-it-ok-to-use-golang-pprof-on-production-without-effecting-performance